kotlin-eclipse-ui/src/org/jetbrains/kotlin/ui/editors/Configuration.java

```diff
@@ -121,11 +125,17 @@ public IReconciler getReconciler(ISourceViewer sourceViewer) {
         return new MonoReconciler(ktReconcilingStrategy, false);
     }
 
+    @Nullable
     protected KotlinTokenScanner getScanner() {
         if (scanner == null) {
-            scanner = new KotlinTokenScanner(EditorUtil.getFile(editor), fPreferenceStore, getColorManager());
+            IFile file = EditorUtil.getFile(editor);
+            if (file != null) {
+                scanner = new KotlinTokenScanner(file, fPreferenceStore, getColorManager());
+            } else {
+                KotlinLogger.log(IStatus.ERROR, "Failed to retrieve IFile from editor " + editor, null);
```

Comment:
Here and further you can use `KotlinLogger.logError(String message, Throwable exception)` method